src/estimator.cpp

```diff
@@ -135,10 +135,12 @@ Estimator::Estimator(const Json::Value &cfg)
 
   triangulate_pre_subfilter_ =
       cfg_.get("triangulate_pre_subfilter", false).asBool();
-  triangulate_options_.method = cfg_["triangulation"].get("method", 1).asInt();
+  triangulate_options_.method = cfg_["triangulation"].get("method", 1).asString();
   triangulate_options_.zmin =
       cfg_["triangulation"].get("zmin", 0.05).asDouble();
   triangulate_options_.zmax = cfg_["triangulation"].get("zmax", 5.0).asDouble();
+  triangulate_options_.max_theta_thresh = cfg_["triangulation"].get("max_theta_thresh", 0.01).asDouble();
```

Comment:
You forgot to actually these parameters in `check_angular_reprojection` and `check_parallax`. :)
Triangulation options are already an input to `Feature::Triangulate`, so the specific parameters just have to go down one level more. (It's okay if the internal interfaces change!)